docs/en/faq/how-kotlin-jdsl-is-different-from-jooq-and-querydsl.md

```diff
@@ -0,0 +1,35 @@
+# How is Kotlin JDSL different from jOOQ and QueryDSL?
+
+Unlike jOOQ and QueryDSL, Kotlin JDSL does not generate a `Metadata Model (code generation) for writing queries.
+
+If you use code generation, you'll have to do the following:
+
+```
+Modify tables and entities →
+Entity modification causes compile errors →
+Run Maven or Gradle task to regenerate the metadata model →
+Write query based on modified entity
+```
+
+Then you might think about trying JPQL yourself, but that might not be a good idea.
+
+JPQL uses a string-based way of writing queries, which makes it impossible to detect errors at compile time.
+
+However, the Kotlin JDSL uses code generation and pure code-based query construction rather than string-based construction.
+
+This means that as soon as you modify an entity or field name, it is reflected in the query code, and the above inconvenience does not exist.
+
+It also adopts an ORM-based, object-oriented query approach like JPQL, which supports specifications such as polymorphism and fetch joins.
+
+Therefore, the way we write queries also relies on object names and fields to query, rather than querying the table/column itself.
+
+Below is a table comparing the differences between Kotlin JDSL, QueryDSL, and jOOQ.
+
+
+
+|                      | Kotlin JDSL | QueryDSL                                       | jOOQ                                           |
+|----------------------|-------------|------------------------------------------------|------------------------------------------------|
+| Code Generation      | ❌           | ✅                                              | ✅                                              | ✅ |
```

Comment:
In my opinion, Code Generation itself is not good, but if it is marked with an X, it looks like JDSL is bad. I think it would be better to create a table that can highlight more advantages.

Reply:
Oh, you are so delicate!

I was thinking the same thing, rightly or wrongly, that the red X emoji would make anything about the benefits look negative.

Reply:
Well, when I was compiling the reviews, I realized that there were a few unnatural things that I wanted to exclude, such as the following. T.T

<img width="798" alt="image" src="https://github.com/line/kotlin-jdsl/assets/66164361/c5f00b5b-efde-47a9-923d-9258ace55e74">